internal/auth0/action/resource.go

```diff
@@ -89,11 +89,10 @@ func NewResource() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					"node12",
 					"node16",
-					"node18",         // Node 18 beta.
-					"node18-actions", // Node 18 GA.
+					"node18-beta",
+					"node18",
 				}, false),
-				Description: "The Node runtime. Defaults to `node12`. Possible values are: " +
-					"`node12`, `node16` or `node18-actions`.",
+				Description: "The Node runtime. Defaults to `node16`. Possible values are:  `node12`, `node16`, `node18-beta` or `node18`.",
```

Comment:
**Recommendation:** Perhaps remove `node12` and `node18-beta` or at the very least note that they are deprecated in the docs.